doc/ipad.md

```diff
@@ -63,3 +64,11 @@ name to use, you can use the Servediter iPad App instead!
 Download [Serveediter](https://apps.apple.com/us/app/servediter-for-code-server/id1504491325) from the
 App Store and then input your server information. If you are running a local server or mabye a usb-c
 connected Raspberry Pi, you will input your settings into "Self Hosted Server".
+
+### Raspberry Pi USB-C Network
+
+It is a bit out of scope for this project, however, great success is being reported using iPad on the go with just a single USB-C cable connected to a Raspberry Pi both powering and supplying direct network access. Many support articles already exist but the key steps boil down to turning on Network over USB-C on the Raspberry Pi itself and the rest of the steps are just like getting Code Server running any where else.
+
+> Here are my keys to success. I bought a 4" touch screen with fan included that attaches as a case to the Pi. I use the touch screen for anytime I have connection issues, otherwise I turn off the Pi screen. I gave my Pi a network name so I can easily connect at home on wifi or when on go with 1 usb-c cable supplying power and network. LASTLY, not all usb-c cables are equal and not all will work so try different usb-c cables if you are going mad (confirm over wifi first then move to cable).
+>
+> -- <cite>Acker Apple</cite>
```

Comment:
Thoughts on linking to your GitHub or email or something else where people may get in touch in case they're trying to follow a similar setup? Or if you have a blog post we can link to, that could work as well?

Reply:
Yes I will do this. Notes to self:

- Add link to my github http://github.com/ackerapple
- Consider documenting the little things to get right when using Magic Keyboard
  - People complain about track pad scrolling issues, I manage just fine
  - Some keyboard short cuts are odd. Like new file, search all, and few others
  - How i use multi task mode to make changes and see browser at same time
  - How using the debug/terminal in VS code is a must so you don’t need another terminal app
- Consider linking the YouTube videos that taught me, I have “liked” them all so should be easy
- Consider adding topic of “Projections by 2022 for iPad replaced by Arm Macs”
  - Large proof Mac arm processors will have touch screens and so the hurdles of coding on iPad today will most likely disappear
- In another PR consider documenting code-server experience as a NodeJs coder
  - Installing was so much easier with `npm install -g code-server` than any other install

Reply:
Sweet! When it's ready for the second review, just re-request it from me and I'll get this merged in 😄

Reply:
@jsjoeio I have finished my edits. Please feel free to make additional suggestions or flat out edits. I trust your judgement and even need to omit for any internal reasons.

Thank you humbly